### DISCUSSION TITLE
X Y Instant Reading

### DISCUSSION THREAD

**Discussion by artisancoffee** [artisan-roaster-scope/artisan#878] [General]
Hello, somehow my setting changed for where you have the mouse.  It used to read x=time y=temp.  Not it read x=time y=ROR.  How do I change it back? Screenshot circled in red what I'm talking about
<img width="1459" alt="Screen Shot 2022-05-06 at 10 05 35 AM" src="https://user-images.githubusercontent.com/105062243/167150128-b957e316-9b34-4568-820f-e11f4d2d1eac.png">
.

**Comment by MAKOMO** [comment]
You are using an outdated version of Artisan! Current version is v2.6.0. The curser widget has been massively improved in that new version. See section [Curser Widget](https://artisan-roasterscope.blogspot.com/2022/03/artisan-v260.html) for details.

You can toggle between temperature and RoR axis by clicking the d key in both versions. Under menu `Help >> Shortcuts` you find a list of all keyboard shortcuts.

**Reply by artisancoffee** [reply]
Thats amazing, much appreciated and I love the new design!  Thanks for the quick response!
